### DISCUSSION TITLE
【求助】怎么改名字啊？

### DISCUSSION THREAD

**Discussion by Ailian-Qiao** [share121/inter-knot#439] [其他]
![Mihoyo Launcher Screenshot 2024 07 12 - 07 36 05 21](https://github.com/user-attachments/assets/4bb5b56f-b7d6-4b5e-82d8-7b4ee062df8b)
注册账号的时候随便弄了一个，现在发现改不了了，在线等，急

**Comment by Ailian-Qiao** [comment]
已经会了

**Comment by lbghhj** [comment]
艾莲可以来找我帮忙哦
